### PR TITLE
Shortstories model snippet function

### PR DESCRIPTION
**Pull request by maryamyasaei**

### CODE REVIEW

File: shortstories/models.py [hunk 1]
```diff
@@ -13,3 +13,6 @@ class ShortStory(models.Model):
     body = models.TextField(default= "No story written by the author")
     status = models.CharField(max_length = 1, choices = PUBLICATION_STATUS)
     publication_date = models.DateField()
+
+    def Snippet(self):
```
Comment (hesamd) [inline shortstories/models.py]: Please read the official python [guide](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names), function names should start with lower case, classes will start with upper case. Also

- [ ] Why do you need parentheses around your return statement
- [ ] Could you check if body is None `return ""`

File: shortstories/tests.py [hunk 2]
```diff
@@ -32,3 +32,10 @@ def  test_current_shortstory(self):
         current_shortstory = create_shortstories(author=user_id, status='p', title ="Villete", body="A romance written by Bronte Charlotte", publication_date='2000-08-25')
         response = self.client.get(reverse('shortstories:detail', args=(current_shortstory.id,)))
         self.assertContains(response, current_shortstory.title)
+
+
+class shortstoriesfunction(TestCase):
+    def test_snippet_function(self):
```
Comment (hesamd) [inline shortstories/tests.py]: Can you add one more test?
Please also the following style naming your class and function
https://docs.djangoproject.com/en/2.1/topics/testing/overview/

**Review by hesamd — CHANGES_REQUESTED** [review]

File: shortstories/tests.py [hunk 3]
```diff
@@ -32,3 +32,10 @@ def  test_current_shortstory(self):
...
+        user_id = User.objects.create(id=1)
+        current_shortstory = create_shortstories(author=user_id, status='p', title ="Villete", body="It was the hunter's first time outside Montana. He woke, stricken still with the hours-old vision of ascending through rose-lit cumulus, of houses and", publication_date='2000-08-25')
+        self.assertEqual (len(current_shortstory.body),150)
```
Comment (hesamd) [inline shortstories/tests.py]: This is not the correct test, you should check the snipper not the body.

**Review by hesamd — CHANGES_REQUESTED** [review]

File: shortstories/models.py (hunk 1 above)
Comment (maryamyasaei) [reply shortstories/models.py]: if body is None, it should return default or just " " ?

Comment (hesamd) [reply shortstories/models.py]: just blank.

**Review by hesamd — APPROVED** [review]

**Pull request merged**